gradle.properties

```diff
@@ -1,4 +1,4 @@
-junitVersion=5.8.2
+junitVersion=5.9.0
```

Comment:
As you know, this actually bumps the JUnit version we use under hood. It not just incorporates it in our build (as suggested in #652). From our [contributing guidelines](https://github.com/junit-pioneer/junit-pioneer/blob/main/CONTRIBUTING.md#updating-junit-5):

>As [documented](https://github.com/junit-pioneer/junit-pioneer/blob/main/README.md#dependencies) Pioneer aims to use the lowest JUnit 5 version that supports Pioneer's feature set. At the same time, there is no general reason to hesitate with updating the dependency if a new feature requires a newer version or the old version has a severe bug.

Our code currently only fails with JUnit 5.9 due to test internals, not because we need a certain feature or fix. I'm cool with it, I just wonder if we can and/or want to stay on 5.8.2?

In any case, please also update the build accordingly:

* https://github.com/junit-pioneer/junit-pioneer/blob/main/.github/workflows/build.yml#L121
* https://github.com/junit-pioneer/junit-pioneer/blob/main/.github/workflows/build.yml#L150

Reply:
You are totally right (e.g. #320 was the issue about upgrading to 5.6.0). But I understood your comment in the issue (#652) as a "we need it" and therefore created this PR. I'll check if I can get the build work without updating. If not I would call a vote on next maintainer meeting about either upgrading the Jupiter version or not adding it to the build matrix to avoid failing builds.

P.S. Will update the two sections of the workflow script you've linked while doing this.

Reply:
Okay in 5.9.0 the interface returns `org.junit.jupiter.api.extension.ExecutableInvoker`, but in 5.8.2 the class is placed `org.junit.jupiter.engine.execution.ExecutableInvoker`. So even if I "overwrite" the `getExecutableInvoker` Methode it would return the false interface.